packages/docs/docs/adlibs/index.md

```diff
@@ -0,0 +1,20 @@
+---
+sidebar_position: 4
+---
+
+# AdLibs
+
+AdLibs are pieces or actions that can be taken by the user during a show. Sofie provides two different types of AdLibs:
+
+1. AdLib Pieces
+2. AdLib Actions
+
+Both can have two context, either belong to a segment/part or have a global context (which is still limited to the Rundown they are in! It's important to keep in mind that a Global AdLib will not be available for an entire RundownPlaylist, it's tied to a given Rundown in that playlist.)
+
+AdLib Pieces are the old way of improvising during a show, they simply insert a piece into it's layer.
```

Comment:
```suggestion
AdLib Pieces are the simple way of improvising during a show, they simply insert a piece into it's layer.
```

While it is older, it is kept around because of the simplicity of it. So I dont think we need to highlight its age, as that suggests that it might be deprecated soon and I dont think that is likely to happen